### DISCUSSION TITLE
DEPRECATION: rename `--policy-*` flags

### DISCUSSION THREAD

**Discussion by knqyf263** [aquasecurity/trivy#6478] [Announcements]
## Summary
Please migrate the following commands:

1. `--policy-bundle-repository` to `--checks-bundle-repository`
2. `--reset-policy-bundle` to `--reset-checks-bundle`
3.  `--policy-namespaces` to `--check-namespaces` 
4. `--config-policy` to `--config-check`
5. `--skip-policy-update` to `--skip-check-update`

## Description
To simplify naming conventions around Rego checks the flags are being renamed to use the convention of calling each Rego check as a "check" instead of the old naming convention of using a "policy".

To facilitate a smooth transition, the old flags will remain operational for a certain period. However, the recommendation is to adopt the new flags for the future use of Trivy.

**Comment by nikpivkin** [comment]
Related issue: https://github.com/aquasecurity/trivy/issues/5609
